python/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ declare_mlir_python_sources(TorchMLIRPythonSources.PublicAPI
   ADD_TO_PARENT TorchMLIRPythonSources
   SOURCES
     fx.py
+    extras/decomp_util.py
```

Comment:
The "extras" namespace is global across a number of compilers. Need to fully qualify things in it. Please rename to `fx_decomp_util.py`.